test_spec_PY3

```diff
@@ -0,0 +1,6 @@
+# Excludes
```

Comment:
Change PY3 to SKA3_HEAD.  This is specifically targeted to run on HEAD machines right?